tezos-sig-gating-contracts/scripts/tests/gatednftminterSimpleMultisig.spec.ts

```diff
@@ -24,19 +24,27 @@ import {
 import { buildTxCallDataNoFunctionName } from "../utils/buildTxCallData";
 import { computePayloadHash } from "../utils/computePayloadHash";
 
-const RPC_ENDPOINT = "http://localhost:20000/";
+import {
+  NEXERAID_SIGNER_SK,
+  DEPLOYER_SK,
+  DEPLOYER_PK,
+  USER_1_PK,
+  USER_2_SK,
+  USER_2_PK,
+} from "./testAddresses";
+
+const RPC_ENDPOINT = "http://localhost:8732/";
 
 const Tezos = new TezosToolkit(RPC_ENDPOINT);
 import { RpcClient } from "@taquito/rpc";
 const client = new RpcClient(RPC_ENDPOINT); //, 'NetXnofnLBXBoxo');
 
-let alice_pk = "edsk3QoqBuvdamxouPhin7swCvkQNgq4jP5KZPbwWNnwdZpSpJiEbq";
+let alice_pk = DEPLOYER_SK;
 let alice = "tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb";
-let bob_pk = "edsk3RFfvaFaxbHx8BMtEW1rKQcPtDML3LXjNqMNLCzC3wLC1bWbAt";
+let bob_pk = NEXERAID_SIGNER_SK;
 let bob = "tz1aSkwEot3L2kmUvcoxzjMomb9mvBNuzFK6";
-let frank_pk =
-  "edskS7YYeT85SiRZEHPFjDpCAzCuUaMwYFi39cWPfguovTuNqxU3U9hXo7LocuJmr7hxkesUFkmDJh26ubQGehwXY8YiGXYCvU";
-let frank = "tz1TiFzFCcwjv4pyYGTrnncqgq17p59CzAE2";
+let frank_pk = USER_2_SK;
+let frank = USER_2_PK;
```

Comment:
if these users have specific roles in the tests such as "multiSigOwner" or "txSigner", I'd rather name them like that.

Reply:
agreed